modules/client-configuration/pages/cobbler.adoc

```diff
@@ -3,6 +3,13 @@
 
 Cobbler is an installation server that allows you to perform unattended system installations.
 It can be used on server, client, or guest systems, and in virtual environments.
+On {productname} Cobbler is installed on the server.
```

Comment:
```suggestion
Cobbler is installed on the {productname} Server.
```

Reply:
Sounds better, but then we'd better remove the sentence before to make the message clear.